src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -277,6 +277,49 @@ public TimeSpan Expect100ContinueTimeout
             }
         }
 
+        public TimeSpan KeepAlivePingDelay
+        {
+            get => _settings._keepAlivePingDelay;
+            set
+            {
+                if ((value < TimeSpan.Zero && value != Timeout.InfiniteTimeSpan) ||
+                    (value.TotalMilliseconds > int.MaxValue))
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value));
+                }
+
+                CheckDisposedOrStarted();
+                _settings._keepAlivePingDelay = value;
+            }
+        }
+
+        public TimeSpan KeepAlivePingTimeout
```

Comment:
Starting point for docs:

https://github.com/dotnet/aspnetcore/blob/29ceed2b2a6bcd9f3064f51cc78858e22c8544c4/src/Servers/Kestrel/Core/src/Http2Limits.cs#L174-L183